app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -43,7 +44,6 @@ description: |
 echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
 
 <% end -%>
-<%= snippet_if_exists(template_name + " custom snippet") %>
```

Comment:
Why to remove this?

Reply:
because I added 2 more snippet hooks.

Reply:
Yeah but removing this could potentially break the workflow for the users that have `custom snippet` with logic in this place.

Reply:
ok. If you want, I can let it as it is and not remove this.